src/components/TermBox.vue

```diff
@@ -1,19 +1,20 @@
 <template>
-	<div class="wikibase-termbox-primaryLanguage">
+	<!-- TODO dynamize wikibase-termbox--primaryLanguage for T205261 -->
+	<div class="wikibase-termbox wikibase-termbox--primaryLanguage">
```

Comment:
'wikibase-termbox' - depends where the other components are located should be probably should be move one level up.

Reply:
I feel the components should be self-contained and found the **B**EM block class present only with a modifier attached problematic.
Compare "Under the hood" http://getbem.com/introduction/